Add signup link AY20/21 S1 (#354)
nushackers/nushackers-site: content/join-coreteam.md

## Want to join us?

> You can sign up using this [form](https://forms.gle/GfYbNgFYdEeBHA8DA).

Reviewer: Can you add the PDPA thing to the form?

Author: ohno

Author: ok done boss @chrisgzf can you take a look

Reviewer: okie i added a separate qn to ensure that people acknowledge should be okay alr

> Following that, we'll set up an interview to find out more about what you're interested in doing. This is usually an informal chat over lunch, so don't worry too much about it; it does help to take part in our events and check out the rest of this site beforehand, though. This is also an opportunity for you to get to know us better, so feel free to ask any questions, too.

Reviewer: ```suggestion
Depending on your responses, we might set up an interview to find out more about what you're interested in doing. This is usually an informal chat over lunch, so don't worry too much about it; it does help to take part in our events and check out the rest of this site beforehand, though. This is also an opportunity for you to get to know us better, so feel free to ask any questions, too.
```
